web-app/react-app/components/ProfilePage/IdentityHandler/web3.js

```diff
@@ -0,0 +1,13 @@
+// @flow
```

Comment:
Just noticed that this script doesn't notice if the account is changed (or metamask is unlocked). Maybe this is something @jtakalai could help to develop?
Either it could notice it through some event (and change the defaultAccount) or we need to use the function `ownWeb3.eth.getAccounts()` in the actions (I think that even `ownWeb3.eth.accounts` doesn't update on account change

Reply:
There's a hack for it, but the official docs say poll with setTimeout  /: